chain/starknet/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "graph-chain-starknet"
+version.workspace = true
+edition.workspace = true
+
+[build-dependencies]
+tonic-build = { workspace = true }
+
+[dependencies]
+graph = { path = "../../graph" }
+hex = { version = "0.4.3", features = ["serde"] }
+prost = { workspace = true }
+prost-types = { workspace = true }
+serde = "1.0"
+starknet-core = "0.6.0"
```

Comment:
Could we get away with using starknet-ff and duplicating the util function? Would be great if we could minimize the amount of deps brought in

Reply:
Since we're calculating hashes we will also need `starknet-crypto`. I will update this part too.

Reply:
Updated with code copied from `starknet-rs`.